### ISSUE TITLE
Live/dynamic and multi period documentation

### ISSUE THREAD

**Issue by patrickkunka** [videojs/mpd-parser#78]
Thanks for open sourcing this. I've managed to get a basic VOD player playing DASH content from mpd parser input. My next challenge will be live content and multi period content. Are these supported and how are these represented in your schema ?

It would be really helpful to have some basic documentation around your schema explaining how each property relates to DASH XML input.

Thanks

**Comment by warren-bank** [comment]
I have a related question..

Is there any attribute in the resulting data structure to distinguish `<MPD type="static"` from `<MPD type="dynamic"`?

examples:
* [static](https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd)
* [dynamic](http://livesim.dashif.org/livesim/testpic_2s/Manifest.mpd)

I'm looking at the JSON that was generated for each, and I'm not seeing any way to determine that one is VOD and the other is not. In fact, both include a `minimumUpdatePeriod` attribute, which (please correct me if I'm wrong) I believe should not be present in a "static" manifest.

**Comment by gesinger** [comment]
Good idea on creating more documentation around that @patrickkunka . We will look into adding some. Right now, to determine if the result is live vs VOD, you'd have to check for a truthy `endList` value on a playlist. This is because we've based the parsed object off of the [m3u8-parser](https://github.com/videojs/m3u8-parser) and https://tools.ietf.org/html/draft-pantos-http-live-streaming-23#section-4.3.3.4

**Comment by warren-bank** [comment]
I thought that attribute might be revealing, but for both of the manifests that I referenced (above).. the resulting data structures contain: `"endList": true`. Can you think of any reason that the "dynamic" dash manifest would also yield this value in the output?

**Comment by gesinger** [comment]
@warren-bank do you have a sample we can look at?

**Comment by warren-bank** [comment]
Just did a quick skim of the code.. and [this](https://github.com/videojs/mpd-parser/blob/cd270036504aa3460ee309f681f392839aa760fb/src/toM3u8.js#L262) implies that the attribute value might be hard coded?

**Comment by warren-bank** [comment]
I was using [this dynamic](http://livesim.dashif.org/livesim/testpic_2s/Manifest.mpd).. mainly because when I was looking for an example, I noticed that some of your samples use it.

**Comment by warren-bank** [comment]
Oh ok.. an `endList` attribute occurs at different levels in the JSON.. each stream has its own. So I just need to get the value from the video stream.. not the top-level value. Gotcha..

**Comment by warren-bank** [comment]
Off-topic question, but it's a quick one.. I'm curious about the name "toM3u8".. is there any functionality to generate HLS (m3u8) manifests (master and children) from a given DASH (mpd) input?

**Comment by gkatsev** [comment]
Our internal JSON representation is sort of based off of HLS, and so we convert the MPD to that same JSON representation. We don't currently have a project that takes that JSON representation and produces an HLS or DASH output.

**Comment by warren-bank** [comment]
Yes, that's what I thought. On the one-hand, it would've been great if that functionality was included for free. On the other hand, I would've felt _really_ dumb.. because the [project](https://github.com/warren-bank/DASH-to-HLS) in which I was using `mpd-parser` at the time was to do just that. (PS: seems to work great)
